app/views/lookup/common/no_result_warning.scala.html

```diff
@@ -72,7 +73,12 @@ <h1 id="no-results-search-term" class="govuk-heading-xl">@Messages(message+ ".su
     @checkURN
 }
 @checkURN = {
-  <a class="button margin-top button--spaced govuk-button" href="check-the-awrs-register" role="button">@Messages("awrs.lookup.check_urn")</a>
+    <div class="govuk-form-group">
```

Comment:
Do we really need this div class - tried without and button appears in the same place and no spacing issues so seems a though it is just additional wrapping of the component

Reply:
Removed as per comment - resolved